### PR TITLE
6 new faculty added to Augusta University. Two middle initials added …

### PR DESCRIPTION
**Pull request by gagrawal28**
…to match dblp enttries

# Contributing to CSrankings

Thanks for contributing to CSrankings! Please read and indicate you agree with **all** these guidelines to getting your pull request accepted. Note that pull requests may take some time to get merged (up to three months, as processing has moved to a quarterly cadence).

**If you find CSrankings useful, [please consider becoming a sponsor](https://github.com/sponsors/csrankings).**

**NOTE:** IF YOU DO NOT FOLLOW THE STEPS BELOW, YOUR COMMIT MAY BE SUMMARILY REJECTED.

1. _Use a reasonable title that explains what the PR corresponds to (as in, not "Update csrankings-x.csv")._

1. _Do not modify any files except `csrankings-[a-z].csv` or (if needed) `country-info.csv`._

1. _Do not use Excel to edit any .csv files; Excel incorrectly tries to
convert some Google Scholar entries to formulas, corrupting the
database. Use a text editor like emacs or NotePad instead._

1. _Insert new faculty **in alphabetical order** (not at the end) in the appropriate `csrankings-[a-z].csv` files._ **Do not modify `csrankings.csv`, which is auto-generated.**

1. _Check to make sure that you have no spaces after commas, or any missing fields._

1. _Check to make sure the home page is correct._

1. _Make sure the Google Scholar IDs are just the alphanumeric identifier (not a URL or with `&hl=en`)._

1. _Check to make sure the name corresponds to the DBLP entry (look it up at http://dblp.org)._

1. _If a faculty member is not in a CS department or similar, include a comment explaining how they meet the inclusion criteria (see below)._

1. _Read and check **all** the boxes below by filling them in with an X._

**Inclusion criteria**

- [ ] Make sure that any faculty you add meet the inclusion
criteria. Eligible faculty include only full-time, tenure-track research
faculty members on a given campus who can *solely* advise PhD students in
Computer Science. Faculty not in a CS department or similar who can
advise PhD students in CS can be included regardless of their home
department. Faculty should also have a 75%+ time appointment (check
`old/industry.csv` for faculty who are now more than 25% in industry).

**Updating an affiliation or home page**

- [ ] Update affiliations, home pages, and Google Scholar entries by modifying `csrankings-[a-z].csv`. For the Google Scholar entry, just use the alphanumeric identifier in the middle of the URL. If none is there, put `NOSCHOLARPAGE`.

**Adding one or more faculty members (including an entire department)**

- [ ] If the department is not yet listed in CSrankings, the entire CS faculty needs to be added (not just one faculty member).

- [ ] Enter each faculty member's [DBLP](http://dblp.org) name, home page, and Google Scholar entry (just the alphanumeric identifier, not the whole URL) by modifying `csrankings-[a-z].csv` (**the letters correspond to the first letter of the faculty members' names**); include disambiguation suffixes like `0001` as needed. If the faculty entry is currently ambiguous, please do not include them. Send mail to the DBLP maintainers (dblp@dagstuhl.de) with a few publications by a particular faculty member; also, open an issue so that when the DBLP database is updated, that faculty member's information can be added.

- [ ] If DBLP has multiple entries for this person, *all of them need to be listed*. Do not update `dblp-aliases.csv`.

- [ ] If the institution you are adding is not in the US,
update `country-info.csv` and add *all* of the faculty in the CS department.

**(Advanced) Quick contribution via a shallow clone** 

We recommend that you use the GitHub web user interface to make changes. However, it may be more convenient
to clone the repository for larger-scale changes.

However, a full clone of the CSrankings repository is almost 2GB.  To
contribute a change without creating a full local clone of the
CSrankings repo, you can perform a _shallow clone_.  To do so, follow these
steps:

1. Fork the CSrankings repo.  If you have an existing fork, but it is not up to date with the main repository, this technique may not work.  If necessary, delete and re-create your fork to get it up to date.  (Do **not** delete your existing fork if it has unmerged changes you want to preserve!)

1. Do a shallow clone of your fork: `git clone --depth 1 https://github.com/yourusername/CSrankings`.  This will only download the most recent commit, not the full git history.

1. Make your changes on a branch, push them to your clone, and create a pull request on GitHub as usual.

If you want to make another contribution and some time has passed, perform these steps again, creating a fresh fork and shallow clone.

### CODE REVIEW

File: csrankings-a.csv [hunk 1]
```diff
@@ -331,7 +331,7 @@ Ahmad Sharieh,University of Jordan,http://eacademic.ju.edu.jo/sharieh,lXeTzzAAAA
 Ahmad-Reza Sadeghi,TU Darmstadt,https://www.trust.informatik.tu-darmstadt.de/people/ahmad-reza-sadeghi,p66Ie-YAAAAJ
 Ahmed Abdelali,Qatar Computing Research Institute,http://www.qcri.org/our-people/bio?pid=103&amp;par=acc&amp;name=AhmedAbdelali,ib1BYugAAAAJ
 Ahmed Abu-Hajar,Washington State University,https://school.eecs.wsu.edu/people/faculty/ahmed-abu-hajar,NOSCHOLARPAGE
-Ahmed Aleroud,Augusta University,https://www.augusta.edu/faculty/directory/view.php?id=AALEROUD,vW6NVIQAAAAJ
+Ahmed F.  Al-Eroud,Augusta University,https://www.augusta.edu/faculty/directory/view.php?id=AALEROUD,vW6NVIQAAAAJ
```
Comment (emeryberger) [inline csrankings-a.csv]: There are three aliases listed for this author in DBLP, and all three need entries (FWIW, the extra space after the "F." will cause problems too). See https://dblp.org/pid/85/9730.html.

File: csrankings-a.csv [hunk 2]
```diff
@@ -112,6 +112,7 @@ Abdul Serwadda,Texas Tech University,http://www.depts.ttu.edu/CS/faculty/faculty
 Abdulaziz Ali,Qatar University,http://www.qu.edu.qa/engineering/computer/faculty/abdulla_alali.php,S42RixYAAAAJ
 Abdulhakim Qahtan,Utrecht University,https://www.uu.nl/staff/AAAQahtan,8eKzXa8AAAAJ
 Abdulla K. Al-Ali,Qatar University,http://www.qu.edu.qa/engineering/computer/faculty/abdulla_alali.php,s1LawbAAAAAJ
+Abdullah Al-Mamun,Augusta University,https://www.augusta.edu/faculty/directory/view.php?id=AALMAMUN,PpBDA1cAAAAJ
```
Comment (emeryberger) [inline csrankings-a.csv]: Please add appropriate disambiguation suffix: https://dblp.org/pid/74/3808.html.

File: csrankings-a.csv [hunk 3]
```diff
@@ -738,7 +739,7 @@ Alexander S. Kotov,Wayne State University,http://www.cs.wayne.edu/kotov,83gsCFsA
 Alexander Schill,TU Dresden,http://www.rn.inf.tu-dresden.de,TABb2iQAAAAJ
 Alexander Schlaefer,Hamburg University of Technology,https://mtec.et8.tuhh.de/staff/alexander-schlaefer.html,NOSCHOLARPAGE
 Alexander Schliep,Chalmers/GU,https://schlieplab.org,NOSCHOLARPAGE
-Alexander Schwarzmann,Augusta University,https://www.augusta.edu/faculty/directory/view.php?id=ASCHWARZMANN,gQo0AHYAAAAJ
+Alexander A. Schwarzmann,Augusta University,https://www.augusta.edu/faculty/directory/view.php?id=ASCHWARZMANN,gQo0AHYAAAAJ
```
Comment (emeryberger) [inline csrankings-a.csv:754]: Please add entry for other alias (see https://dblp.org/pid/s/AlexanderAShvartsman.html).

**Review by emeryberger — CHANGES_REQUESTED** [review]
See comments.

**Comment by gagrawal28** [comment]
Thanks for the feedback, I think I have fixed each of these issues.

File: csrankings-a.csv [hunk 4]
```diff
@@ -336,7 +336,9 @@ Ahmad Sharieh,University of Jordan,http://eacademic.ju.edu.jo/sharieh,lXeTzzAAAA
 Ahmad-Reza Sadeghi,TU Darmstadt,https://www.trust.informatik.tu-darmstadt.de/people/ahmad-reza-sadeghi,p66Ie-YAAAAJ
 Ahmed Abdelali,Qatar Computing Research Institute,http://www.qcri.org/our-people/bio?pid=103&amp;par=acc&amp;name=AhmedAbdelali,ib1BYugAAAAJ
 Ahmed Abu-Hajar,Washington State University,https://school.eecs.wsu.edu/people/faculty/ahmed-abu-hajar,NOSCHOLARPAGE
-Ahmed F.  Al-Eroud,Augusta University,https://www.augusta.edu/faculty/directory/view.php?id=AALEROUD,vW6NVIQAAAAJ
+Ahmed F.Al-Eroud,Augusta University,https://www.augusta.edu/faculty/directory/view.php?id=AALEROUD,vW6NVIQAAAAJ
```
Comment (emeryberger) [inline csrankings-a.csv]: There needs to be exactly one space between F. and Al

File: csrankings-s.csv [hunk 5]
```diff
@@ -580,7 +580,7 @@ Sema Fatma Oktug,Istanbul Technical University,http://web.itu.edu.tr/oktug,3TRcR
 Sema Oktug,Istanbul Technical University,http://web.itu.edu.tr/oktug,3TRcRPkAAAAJ
 Semih Salihoglu,University of Waterloo,https://cs.uwaterloo.ca/~ssalihog,XxQq8nsAAAAJ
 Sen Cheng,Ruhr-University Bochum,https://www.ini.rub.de/the_institute/people/sen-cheng,_dzbEm4AAAAJ
-Sen He,Augusta University,https://www.augusta.edu/faculty/directory/view.php?id=SEHE,ySPHD4cAAAAJ
+Sen He,Augusta University,https://www.augusta.edu/ccs/faculty.php,SPHD4cAAAAJ
```
Comment (emeryberger) [inline csrankings-s.csv]: Please restore the original home page (https://www.augusta.edu/faculty/directory/view.php?id=SEHE).

**Comment by emeryberger** [comment]
Two more issues.

**Comment by gagrawal28** [comment]
hopefully this time all is good!

**Pull request merged**